src/fides/api/app_setup.py

```diff
@@ -185,14 +169,8 @@ async def run_database_startup(app: FastAPI) -> None:
     finally:
         db.close()
 
-    load_default_experience_configs()  # Must occur before loading default privacy notices
-
     if not CONFIG.test_mode:
-        # Default notices subject to change, so preventing these from
-        # loading in test mode to avoid interfering with unit tests.
-        load_default_privacy_notices()
```

Comment:
Out of the box configs have moved to Plus.  Previously Experience Configs were loaded first, but now that Notices are attached to Experience Configs, in plus, we flip this order.